os/customizing-sshd.md

```diff
@@ -143,3 +143,73 @@ Enter passphrase for key '/home/user/.ssh/id_rsa':
 ### Further reading on systemd units
 
 For more information about configuring CoreOS hosts with `systemd`, see [Getting Started with systemd]({{site.baseurl}}/docs/launching-containers/launching/getting-started-with-systemd/).
+
+### Override socket-activated SSH
+
+Occasionally when systemd gets into a broken state, socket activation doesn't work, which can make a system inaccessible if ssh is the only option. This can be avoided configuring a permanently active SSH daemon that forks for each incoming connection.
+
+We start by creating a unit to override the default sshd unit file at `/etc/systemd/system/sshd.service`:
+
+```
+# /etc/systemd/system/sshd.service
+[Unit]
+Description=OpenSSH server daemon
+
+[Service]
+Type=forking
+PIDFile=/var/run/sshd.pid
+ExecStart=/usr/sbin/sshd
+ExecReload=/bin/kill -HUP $MAINPID
+KillMode=process
+Restart=on-failure
+RestartSec=30s
+
+[Install]
+WantedBy=multi-user.target
+```
+
+Next we mask the systemd.socket unit.
+```
+# systemctl mask sshd.socket
+```
+Finally, execute a daemon-reload, stop the ssh.socket service, and start the sshd.service unit:
+
+```
+# systemctl daemon-reload
+# systemctl stop sshd.socket; systemctl restart sshd.service
+```
+
+This process via cloud-config:
```

Comment:
The same configuration can be achieved and an actively listening sshd started by providing a cloud-config like: